lib/SparqlEndpointFetcher.ts

```diff
@@ -1,8 +1,8 @@
-import { Readable } from 'node:stream';
-import type { ReadableStream as ReadableWebStream } from 'node:stream/web';
+import type { Readable } from 'node:stream';
```

Comment:
Can we use `@types/readable-stream` instead?

Reply:
Oh actually, maybe fewer changes would work.

Reply:
I have changed the code to use the `Readable` type from `readable-stream` over Node's own types. It seems to pass the tests, so unless someone uses a very exotic function or property that exists in one but not the other, it should work I think.